src/builder/module.rs

```diff
@@ -85,7 +86,7 @@ impl From<elements::Module> for ModuleScaffold {
 			element: element.unwrap_or_default(),
 			code: code.unwrap_or_default(),
 			data: data.unwrap_or_default(),
-			other: sections,
+			other,
```

Comment:
`sections` is always empty at that point. The old code makes no sense. Good spot.